clang/test/CodeGenSYCL/address-space-new.cpp

```diff
@@ -6,6 +6,16 @@ void test() {
   // CHECK-LEGACY: @_ZZ4testvE3foo = internal constant i32 66, align 4
   // CHECK-NEW:    @_ZZ4testvE3foo = internal addrspace(1) constant i32 66, align 4
 
+  // Intentionally leave a part of an array uninitialized. This triggers a
```

Comment:
Does fully initialized array have the same problem?

Reply:
No.